cmux-tui/frontends/web/src/styles.css

```
@@ -42,7 +45,9 @@
   --ansi-bright-white: #ffffff;
   color: var(--terminal-foreground);
   background: var(--terminal-background);
-  font-family: "SFMono-Regular", Consolas, "Liberation Mono", monospace;
+  font-family: Menlo, "SFMono-Regular", Consolas, "Liberation Mono", monospace;
```

Stylelint (17.14.0)

[error] Expected no quotes around "SFMono-Regular" (font-family-name-quotes)

(font-family-name-quotes)